src/Http/Headers/src/SetCookieHeaderValue.cs

```diff
@@ -561,18 +561,34 @@ private static int GetSetCookieLength(StringSegment input, int startIndex, out S
                     return 0;
                 }
 
+                offset = offset + HttpRuleParser.GetWhitespaceLength(input, offset);
```

Comment:
Why was this whitespace check added?

Reply:
I added it in case of having a space before the minus sign.

Reply:
Let's not for now. That's against spec and we don't have any examples of it being a problem.

Reply:
done